rebar.config

```diff
@@ -4,7 +4,7 @@
 
 {deps,
     [{webmachine, ".*",
-       {git, "git://github.com/opscode/webmachine", {tag, "1.10.5.3"}}}
+       {git, "git://github.com/basho/webmachine", {tag, "1.10.8"}}}
```

Comment:
we could, but maybe still err on the side of caution when pointing at external deps